src/style.css

```diff
@@ -0,0 +1,121 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+:root {
+  --color-background-buy-card: #00acdc;
+}
+
+.page {
+  font-family: "Roboto", sans-serif;
+  color: #060b35;
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  height: 406px;
+  width: 198px;
```

Comment:
do not hardcode height for card
it should be stretched by content
also consider using [`box-sizing: border-box`](https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing) for all elements, instead of subtracting borders